// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

syntax = "proto2";

// For explanation, see include/srsran/scheduler/scheduler_feedback_handler.h
message mac_sched_crc_indication {
    required uint64 timestamp = 1;
    required uint32 ue_index = 2;
    required uint32 harq_id = 3;
    required uint32 tb_crc_success = 4;
    required uint32 ul_sinr_dB = 5;
    required uint32 ul_rsrp_dBFS = 6;
    required uint32 time_advance_offset = 7;
}





